### DISCUSSION TITLE
Battery Profiling

### DISCUSSION THREAD

**Discussion by tcamise-gpsw** [gopro/OpenGoPro#35] [General]
This is a pretty open-ended investigation for now of topics spawned from https://github.com/gopro/OpenGoPro/issues/19. The idea is to use the battery logger demo from the Python SDK to do:
- general profiling of battery. Does it decrease in a sensible manner?
- how does enabling / disabling WiFi affect this?
- Ultimately, is there anything that needs to be changed in firmware?

**Comment by tcamise-gpsw** [comment]
I have some initial feedback here. After discussing internally, we do not expect turning off WiFi to show a noticeable difference in the power consumption. Some much bigger factors are going to be, for example, whether the front and / or back screens are on.

I hope to start running actual tests soon to show this.

**Comment by KonradIT** [comment]
I noticed a noticeable battery increase (+~30 mins) when running a GoPro HERO7 (video recording) with no WiFi as opposed to WiFi + BLE.

**Comment by tcamise-gpsw** [comment]
That's interesting. It's possible that this has changed now. Anyway, I hope to have some data soon.
